CHANGELOG.carto.md

```diff
@@ -1,3 +1,13 @@
+## 2.5.3.2+carto-1
+
+**Release date**: 2019-12-16
+
+Changes:
+- Backport [a4a040c](https://github.com/postgis/postgis/commit/a4a040c4949089f32b63e6a647d1d9696992a132): Avoid throwing during update if st_union(geometry) doesn't exist.
```

Comment:
No need to mention individual backports if they are included in the merge of `stable-2.5`.